content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -147,12 +147,10 @@ kubectl api-resources --namespaced=false
 
 ## Automatic labelling
 
-{{< feature-state for_k8s_version="1.22" state="stable" >}}
+{{< feature-state for_k8s_version="1.22" state="stable" for_k8s_version="1.23" state="removed" >}}
```

Comment:
```suggestion
{{< feature-state for_k8s_version="1.22" state="stable" >}}
```

We didn't remove this feature in v1.23 (we removed the _feature gate_, which is different)